tools/upload_to_model.py

```
@@ -0,0 +1,616 @@
+    python upload_to_model.py --source-file /path/to/file.bin --namespace default --model-name my-model --bearer-token your-token
+
+Requirements:
+    - requests
+    - tqdm (for progress bar)
+"""
+
+import os
+import sys
+import argparse
+import requests
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)